src/org/opendatakit/briefcase/ui/reused/AggregateServerConnectionDialog.java

```diff
@@ -14,25 +15,25 @@
   private boolean isSuccessful;
   private ServerConnectionInfo serverInfo = null;
   private TerminationFuture terminationFuture = new TerminationFuture();
+  private AggregateServerConnectionConfiguration aggregateServerConnectionConfiguration;
 
-  AggregateServerConnectionDialog(AggregateServerConnectionDialogForm aggregateServerConnectionDialogForm, boolean asTarget) {
+  AggregateServerConnectionDialog(AggregateServerConnectionDialogForm aggregateServerConnectionDialogForm, boolean asTarget, AggregateServerConnectionConfiguration aggregateServerConnectionConfiguration) {
```

Comment:
Let's name those long args simply `form` and `configuration` for brevity's sake

Reply:
Ok